apis/Google.Cloud.PubSub.V1/docs/index.md

```diff
@@ -82,76 +82,6 @@ restart listening for messages with `StartAsync(...)` again. Due to the expense
 instance, it is recommended that a singleton client per topic is used for the lifetime of the
 application.
 
-## Dependency Injection
```

Comment:
I'm not sure about this - the AddSubscriberClient and AddPublisherClient methods do have slightly different overloads to the pure-GAPIC ones. I suspect we don't need as much detail as we currently have, but I think some is important - including the aspects about automatic stopping.